app/javascript/packs/application.js

```diff
@@ -7,6 +7,13 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import 'bootstrap'
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
+require("packs/custom")
+require("packs/posts.js")
```

Comment:
What was the reason for adding these 2 lines? 🤔

Reply:
I shouldn't have added them, they weren't necessary for the operation.

Reply:
Gotcha